### PR TITLE
[IMP] Translation information

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 [![Build Status](https://travis-ci.org/OCA/purchase-workflow.svg?branch=8.0)](https://travis-ci.org/OCA/purchase-workflow)
```
Comment (pedrobaeza) [inline README.md:2]: Please add runbot badge

**Comment by pedrobaeza** [comment]
You have to rebase (runbot badge is already added)

File: README.md (hunk 1 above)
Comment (yvaucher) [inline README.md:2]: Isn't runbot status already there ? (may have been added in another PR?)

**Comment by yvaucher** [comment]
to rebase

**Pull request closed without merging**